webui/react/src/types.ts

```diff
@@ -32,11 +32,6 @@ export interface Auth {
   user?: DetailedUser;
 }
 
-export interface Credentials {
```

Comment:
maybe we could leave this in and use this instead

Reply:
I think it's better to rely on generated TS interfaces instead of replicating them, having a "middle" interface looks close to code-replication.

Of course there are case where the argument could be different (composition, ...).